invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/theme.js

```diff
@@ -37,11 +37,19 @@ $tabElement.on("keydown", function (event) {
   }
 });
 
-
+const affiliationsButton = $('.title .affiliations-button');
 $(".ui.accordion.affiliations").accordion({
   selector: {
     trigger: ".title .affiliations-button",
   },
+  onOpening: () => {
+    affiliationsButton.attr('aria-expanded', true);
+    affiliationsButton.html("Hide affiliations");
+  },
+  onClosing: () => {
+    affiliationsButton.attr('aria-expanded', false);
+    affiliationsButton.html("Show affiliations");
```

Comment:
Because it needs to change when opening/closing. If not added here, the text will remain "Hide affiliations". The translation is indeed an issue here, maybe the button text can be added as a data-attribute instead?